lib/libzfsacl/zfsacl/libzfsacl_impl_freebsd.c

```diff
@@ -0,0 +1,583 @@
+/*
+ * CDDL HEADER START
+ *
+ * The contents of this file are subject to the terms of the
+ * Common Development and Distribution License (the "License").
+ * You may not use this file except in compliance with the License.
+ *
+ * You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+ * or https://opensource.org/licenses/CDDL-1.0.
+ * See the License for the specific language governing permissions
+ * and limitations under the License.
+ *
+ * When distributing Covered Code, include this CDDL HEADER in each
+ * file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+ * If applicable, add the following below this CDDL HEADER, with the
+ * fields enclosed by brackets "[]" replaced with your own identifying
+ * information: Portions Copyright [yyyy] [name of copyright owner]
+ *
+ * CDDL HEADER END
+ */
+
+/*
+ * Copyright (c) 2022 Andrew Walker <awalker@ixsystems.com>
+ * All rights reserved.
+ */
+
+#include <zfsacl.h>
+#include "/usr/include/sys/acl.h"
```

Comment:
Should I be worried about this hardcoded header path?  Or is this normal convention for FreeBSD?

Reply:
Unfortunately, there are multiple `sys/acl.h` files in the project, when include as `<sys/acl.h>` or `"sys/acl.h"`, it tends to include those files instead to look under `/usr/include`. I did not find a better way to handle this at the time.

I also tried using `-isystem` in `CFLGAS` in makefile but that did not produce a positive result either.

Reply:
Ok, sounds like there's a good reason for it then, thanks.